tasks/generate-info.js

```diff
@@ -92,6 +99,11 @@ function getNewer(date, newer, callback) {
     callback(new Error('Trouble walking ' + sourceDir));
   });
   walker.on('end', function() {
+  
+    if (isWindows) {
+        paths = [sourceDir].concat(externsPaths);
+    }
+  
```

Comment:
It came from a previous windows build PR here 
https://github.com/openlayers/ol3/pull/2813/files#diff-47e08054189bdf70c82633c1a2479eca
looks like the comment block got lost. I am not sure all the changes in this module are needed anymore. I am running some tests.